linuxbrew-build.sh

```diff
@@ -0,0 +1,135 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+ABS_PATH_LIMIT=85
```

Comment:
Make this a read-only integer constant:
```suggestion
declare -i -r ABS_PATH_LIMIT=85
```
Also put it in a common file sourced into both scripts that use this constant.